### DISCUSSION TITLE
How Fooocus is able to run in colab free tier?

### DISCUSSION THREAD

**Discussion by NaveenSohal** [lllyasviel/Fooocus#2634] [Q&A]
I have used Automatic1111 before colab ban and using Fooocus now except not able to batch generate for making videos and also fooocus is missing extensions, models and LoRa downloads. Fooocus also uses WebUI which is banned in free tier. how fooocus is able to do this. I been trying to do this but i got little to no coding knowledge. Can someone help/explain or even hint me in the right direction.

**Comment by mashb1t** [comment]
Maybe i didn't get your question, but please use the official Colab link to start Fooocus there: https://colab.research.google.com/github/lllyasviel/Fooocus/blob/main/fooocus_colab.ipynb
